metis-schema/src/main/resources/schema_xsds/ADMS.xsd

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema xmlns="http://www.w3.org/2001/XMLSchema"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    targetNamespace="http://www.w3.org/ns/adms#">
+    
+    <annotation>
+        <documentation> Europeana representation of Asset Description Metadata Schema (ADMS) elements</documentation>
+    </annotation>
+    
+    <import namespace="http://www.w3.org/1999/02/22-rdf-syntax-ns#" schemaLocation="RDF.xsd"/>
+
+    <element name="status" type="rdf:LiteralType">
+    </element>
+
+</schema>
```

Comment:
These probably introduce a code duplication in our metis code base, right? Are they the same as the ones in the validation library? Is there any way we can avoid the duplication? May not be so easy... perhaps we can create a ticket to that effect?

Reply:
They might not be identical.
The idea is to remove one of the two and have one source of truth, and there is a ticket to do that in the backlog

Reply:
OK. Let's make sure that that ticket reflects the changes you're making here.